keyboards/skyloong/gk75/q1/config.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef ENCODER_ENABLE
+#define ENCODERS_PAD_A {A1, A3, A0, C14}
+#define ENCODERS_PAD_B { A2, A4, C13, C15 }
+#define ENCODER_RESOLUTIONS { 1, 1, 1, 1 }
```

Comment:
This could be moved to `info.json`:
```json
    "encoder": {
        "rotary": [
            { "pin_a": "A1", "pin_b": "A2", "resolution": 1 },
            { "pin_a": "A3", "pin_b": "A4", "resolution": 1 },
            { "pin_a": "A0", "pin_b": "C13", "resolution": 1 },
            { "pin_a": "C14", "pin_b": "C15", "resolution": 1 }
        ]
    },
```
Also `"resolution": 1` is somewhat questionable (it may work if you actually have such encoders and reliable hardware debouncing for them).

There is no JSON mapping for `ENCODER_MAP_KEY_DELAY` at the moment, so it would need to stay here.

Reply:
Okay, I'll correct it.